master/version/version.go

```diff
@@ -8,3 +8,7 @@ const Unset = "unknown"
 // WARN: if you move it to a different package, you need to change the linked
 // path in the make file and CI.
 var Version = Unset
+
+// IsEE returns true if this is the enterprise codebase. It could parse the version, but the dev branch
+// of EE doesn't provide any differentiation from OSS.
+var IsEE = false
```

Comment:
you will need to set this to true in ```TestLoginRemote```

Reply:
well, did something slightly more complicated, so that it is correct in both cases 
```
	tmp := version.IsEE
	version.IsEE = true
	defer func() { version.IsEE = tmp }()
```